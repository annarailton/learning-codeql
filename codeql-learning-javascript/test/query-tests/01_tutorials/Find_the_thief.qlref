01_tutorials/Find_the_thief.ql