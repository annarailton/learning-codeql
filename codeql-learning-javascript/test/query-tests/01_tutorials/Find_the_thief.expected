WARNING: Unused predicate isBaldWithResult (/Users/annarailton/projects/learning-codeql/codeql-learning-javascript/src/01_tutorials/Find_the_thief.ql:15,9-25)
WARNING: Unused predicate isOldestAggregate (/Users/annarailton/projects/learning-codeql/codeql-learning-javascript/src/01_tutorials/Find_the_thief.ql:35,11-28)
WARNING: Unused predicate isOldestOrderedAggregate (/Users/annarailton/projects/learning-codeql/codeql-learning-javascript/src/01_tutorials/Find_the_thief.ql:47,11-35)
| Hester |
